src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -489,6 +502,9 @@ namespace OpenMS
     /// Drift time
     double drift_time_;
 
+    /// Drift time unit
+    DriftTimeUnit drift_time_unit_;
```

Comment:
@timosachsenberg the question here is whether we want to have `drift_time_` and `drift_time_unit_` as attributes of MSSpectrum or of SpectrumSettings. In one way its more like retention time, so I feel it should be in the spectrum. 

On the other hand, its also unclear why the `native_id_` is in the SpectrumSettings and the `name_` is in the MSSpectrum. Maybe the correct approach is to move all members into SpectrumSettings and only keep data in MSSpectrum?

Reply:
PS: why do we even have a `name_` attribute? It is not set in any of the handlers including mzML, mzXML and mzData

Reply:
please feel free to clean this up and decide for yourself. I would be fine having it in the spectrum.